pkg/operators/oci-handler/oci.go

```diff
@@ -201,6 +209,33 @@ func (o *OciHandlerInstance) ExtraParams(gadgetCtx operators.GadgetContext) api.
 	return o.extraParams
 }
 
+func warnBuilderVersion(manifest *v1.Manifest, logger logger.Logger) {
```

Comment:
```suggestion
func checkBuilderVersion(manifest *v1.Manifest, logger logger.Logger) {
```

"warn*" sounds like it should warn all the time (e.g. check already done, now warn).